app/src/main/java/app/revanced/integrations/youtube/settings/Settings.java

```diff
@@ -52,13 +56,14 @@ public class Settings extends BaseSettings {
     public static final BooleanSetting HIDE_VIDEO_ADS = new BooleanSetting("revanced_hide_video_ads", TRUE, true);
     public static final BooleanSetting HIDE_WEB_SEARCH_RESULTS = new BooleanSetting("revanced_hide_web_search_results", TRUE);
     // Layout
-    public static final BooleanSetting ALT_THUMBNAIL_STILLS = new BooleanSetting("revanced_alt_thumbnail_stills", FALSE);
-    public static final IntegerSetting ALT_THUMBNAIL_STILLS_TIME = new IntegerSetting("revanced_alt_thumbnail_stills_time", 2, parent(ALT_THUMBNAIL_STILLS));
-    public static final BooleanSetting ALT_THUMBNAIL_STILLS_FAST = new BooleanSetting("revanced_alt_thumbnail_stills_fast", FALSE, parent(ALT_THUMBNAIL_STILLS));
-    public static final BooleanSetting ALT_THUMBNAIL_DEARROW = new BooleanSetting("revanced_alt_thumbnail_dearrow", FALSE);
+    public static final EnumSetting<ThumbnailOption> ALT_THUMBNAIL_HOME = new EnumSetting<>("revanced_alt_thumbnail_dearrow_home", ThumbnailOption.ORIGINAL);
+    public static final EnumSetting<ThumbnailOption> ALT_THUMBNAIL_SUBSCRIPTIONS = new EnumSetting<>("revanced_alt_thumbnail_dearrow_subscription", ThumbnailOption.ORIGINAL);
```

Comment:
Are the thumbnails also active on the channel page?

Reply:
> how the user got to the channel page

Perhaps this can be fixed by listening for a component that is only present on a channel page (e.g the banner of a YouTube channel). If that component is seen, it means the user is in a channel and the thumbnails (or other patches that are selectively enabled such as shorts) can be shown or hidden.

Reply:
I can see alternative thumbnails being useful on a channel in particular. Leaving the user to figure out how to circumvent this limitation can almost be a dealbreaker for many. Wouldn't listening for a Litho element render such as the video thumbnail of the home feed work? The home feed path is different from search, history, feed below player and subscriptions for example. Similarly the other pages have different paths for the video thumbnails as far as I know.

Reply:
I'd say when an issue is opened we can look into it.